.github/ISSUE_TEMPLATE/standard-issue.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Standard Issue
+about: Standard information for PASS issues
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please answer the following questions for yourself before submitting an issue. 
+**YOU SHOULD DELETE THE PREREQUISITES SECTION.**
+
+# Prerequisites
+
+- [ ] Is the title clear and understandable at a glance?
+- [ ] Is a label present?
```

Comment:
More detail would be useful here, as projects use labels in different ways. Someone coming to PASS from outside won't know what this means. Are labels required for all issue tickets? What labels are recommended?

Reply:
Maybe something like: "Labels are not required for all tickets, but they are helpful for prioritization. Please check the labels drop-down to see if any of the pre-set labels work for your ticket."

Reply:
Change is a label present to indicate is optional. Include the suggested text.